torch/distributed/_sharded_tensor/ops/allclose.py

```diff
@@ -0,0 +1,7 @@
+import torch
```

Comment:
Probably just have a single file for comparison ops like equal and allclose instead of a file per op.

Reply:
Good point, will merge the two files.